spec/payment_spec.rb

```diff
@@ -51,8 +51,39 @@
     }
   }
 
+  let(:boleto_payment) {
+    {
+      'Address' => 'N/A, 1',
+      'Amount' => 1150,
```

Comment:
Amount is already mapped. :)

I'm not sure what use we would have for BoletoNumber, though. Do you have anything in mind?